ext/curl/curl.stub.php

```diff
@@ -2567,17 +2547,14 @@
  * @cvalue CURL_VERSION_UNIX_SOCKETS
  */
 const CURL_VERSION_UNIX_SOCKETS = UNKNOWN;
-#endif
 
-#if LIBCURL_VERSION_NUM >= 0x072900 /* Available since 7.41.0 */
 /**
  * @var int
  * @cvalue CURLOPT_SSL_VERIFYSTATUS
  */
 const CURLOPT_SSL_VERIFYSTATUS = UNKNOWN;
-#endif
 
-#if LIBCURL_VERSION_NUM >= 0x072a00 /* Available since 7.42.0 */
+/* Available since 7.42.0 */
```

Comment:
Thanks for noticing them, should have indeed removed them. I pushed with the entire `#if LIBCURL...` lines removed including the comments. For Versions >= 7.61, we still keep them. There are some leftover comments like `/* Available since 7.10.6 */`, not sure if we can clean them up at the same go, but I thought not to do it yet to keep the PR easier to (hopefully) merge. If you think we can clean everything up, I'll happily change them too.